girder/taskflow/server/models/taskflow.py

```diff
@@ -61,6 +61,7 @@ def append_to_log(self, taskflow, log):
             }
         }
 
+        send_log_notification('taskflow', taskflow, log)
```

Comment:
I would move it after the update, that way we wouldn't send out a notification if the update failed and the log wasn't really updated.